x-pack/plugin/src/test/resources/rest-api-spec/api/security.clear_api_key_cache.json

```
@@ -18,6 +18,12 @@
               "description":"A comma-separated list of IDs of API keys to clear from the cache"
             }
           }
+        },
+        {
+          "path":"/_security/api_key/*/_clear_cache",
```

```suggestion
          "path":"/_security/api_key/_clear_cache",
```

paths should not encode values e.g`*` for ids. 

Places that accept `*` or `_all` typically have routes without the route parameter (e.g `ids`). 

So the above path needs to exist on the server as well.

As per https://github.com/elastic/elasticsearch/issues/63885 having ids optional was originally intended and so the suggested path should exist in Elasticsearch.

The fix that went in was to explicitly add `*` for `ids`, this fixed the test failures but not the inconsistency.

---

I see, thanks @Mpdreamz - Do we need to wait for the other path to be supported first and raise an issue for that?

---

I spent today updating my stale PR's to elasticsearch so while I was there opened: https://github.com/elastic/elasticsearch/pull/65976 to add these additinal routes to the server too,

---

Thanks @Mpdreamz - Since you've added that PR, I will close this PR